modules/phase_field/include/kernels/ACGrGrMulti.h

```diff
@@ -38,6 +38,7 @@ class ACGrGrMulti : public ACGrGrBase
 
   /// Values of gammas for each order parameter
   std::vector<const MaterialProperty<Real> *> _prop_gammas;
+  const MaterialProperty<Real> & _mu;
```

Comment:
Why is this still here. Larry is right. You are shadowing the base class member. This is not the right design.

Reply:
I see now what you are doing (@laagesen pointed it out). You allow the user to specify a custom name. If the model allows for that the change should be made in the base class, not by shadowing a member. If the model does not allow for that, because it breaks the variational derivative it should not be done at all.

Reply:
After talking to Larry today, I'm taking this out and I will work on implementing this differently in the model.

Reply:
@itgreenquist if you want to do what we talked about on the phone earlier, you still need to have the same mu throughout, and instead you make mu a function of the order parameters. That can be done by making mu a DerivativeParsedMaterial or creating a new class for it. Also you will probably need a new kernel to account for the effect of the variational derivative on the (now order parameter dependent) mu.